torchgeo/datasets/south_africa_crop_type.py

```diff
@@ -0,0 +1,248 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""South Africa Crop Type Competition Dataset."""
+
+import os
+import re
+from typing import Any, Callable, Optional, cast
+
+import matplotlib.pyplot as plt
+import torch
+from matplotlib.figure import Figure
+from rasterio.crs import CRS
+from torch import Tensor
+
+from .geo import RasterDataset
+from .utils import BoundingBox, RGBBandsMissingError
+
+
+class SouthAfricaCropType(RasterDataset):
+    """South Africa Crop Type Challenge dataset.
+
+    The `South Africa Crop Type Challenge
+    <https://beta.source.coop/repositories/radiantearth/south-africa-crops-competition/description/>`__
+    dataset includes satellite imagery from Sentinel-1 and Sentinel-2 and labels for
+    crop type that were collected by aerial
+    and vehicle survey from May 2017 to March 2018. Data was collected by the
+    provided by the Western Cape Department of Agriculture and is
```

Comment:
collected by or provided by? Could probably make the length of each line more uniform, it just needs to be less than 88 chars.